src/main/java/io/github/thebusybiscuit/slimefun4/integrations/OrebfuscatorIntegration.java

```diff
@@ -0,0 +1,63 @@
+package io.github.thebusybiscuit.slimefun4.integrations;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+import javax.annotation.Nonnull;
+
+import org.bukkit.Bukkit;
+import org.bukkit.block.Block;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.EventPriority;
+import org.bukkit.event.Listener;
+
+import io.github.thebusybiscuit.slimefun4.api.events.BlockPlacerPlaceEvent;
+import io.github.thebusybiscuit.slimefun4.api.events.ExplosiveToolBreakBlocksEvent;
+import io.github.thebusybiscuit.slimefun4.api.events.ReactorExplodeEvent;
+import io.github.thebusybiscuit.slimefun4.implementation.SlimefunPlugin;
+import net.imprex.orebfuscator.api.OrebfuscatorService;
+
+/**
+ * 
+ * This handles block breaks with orebfuscator
+ * 
+ * @author NgLoader
+ *
+ */
+public class OrebfuscatorIntegration implements Listener {
+
+	private final SlimefunPlugin plugin;
+	private OrebfuscatorService service;
+
+	public OrebfuscatorIntegration(@Nonnull SlimefunPlugin plugin) {
+        this.plugin = plugin;
+    }
+
+	/**
+	 * Init orebfuscation service and register listener
+	 */
+    public void register() {
+        this.service = Bukkit.getServer().getServicesManager().getRegistration(OrebfuscatorService.class).getProvider();
+
+        Bukkit.getServer().getPluginManager().registerEvents(this, this.plugin);
```

Comment:
Minor detail but if we can get the `Server` instance in an object-oriented fashion, it should be preffered over the static access.

```suggestion
        plugin.getServer().getPluginManager().registerEvents(this, this.plugin);
```